test/java/seedu/addressbook/util/TestUtil.java

```diff
@@ -137,4 +139,11 @@ public static void assertTextFilesEqual(Path path1, Path path2) throws IOExcepti
         List<String> list2 = Files.readAllLines(path2, Charset.defaultCharset());
         assertEquals(String.join("\n", list1), String.join("\n", list2));
     }
+
+    /**
+     * Asserts that the file given does not exist on the filesystem.
+     */
+    public static void assertFileDoesNotExist(String filePath) {
+        assertTrue(Files.notExists(Paths.get(filePath)));
```

Comment:
Is there really a need to put this in TestUtil? Is there a rationale behind this? Because it looks to me it will only be used once.